test/performance/scale_revision_by_load_test.go

```diff
@@ -156,6 +156,7 @@ func scaleRevisionByLoad(t *testing.T, numClients int) []junit.TestCase {
 		BaseQPS:        qpsPerClient * float64(numClients),
 		URL:            fmt.Sprintf("http://%s/?timeout=%d", *endpoint, processingTimeMillis),
 		LoadFactors:    []float64{1},
+		FileNamePrefix: strings.Replace(t.Name(), "/", "_", -1),
```

Comment:
@srinivashegde86  , this is for fixing build error, does this make sense to you?

Reply:
yup.